cli/integration/tests/waiter/test_cli.py

```diff
@@ -447,7 +447,8 @@ def test_delete_single_service(self):
         token_name = self.token_name()
         util.post_token(self.waiter_url, token_name, util.minimal_service_description())
         try:
-            self.logger.info(f'Token: {util.load_token(self.waiter_url, token_name)}')
+            self.logger.info(f'python3'
```

Comment:
`python3`?

Reply:
My bad, removed. The new IntelliJ version has been a pain :(